cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -1061,6 +1061,16 @@ mkP2PArguments nForkPolicy cForkPolicy NodeConfiguration {
         Cardano.PeerSelection.readUseBootstrapPeers = daReadUseBootstrapPeers
       }
 
+-- TODO remove and use defaultSyncTargets for next o-n-release.
+defaultSyncTargets' :: PeerSelectionTargets -> PeerSelectionTargets
+defaultSyncTargets' deadlineTargets =
+  deadlineTargets {
+    targetNumberOfActivePeers               = 0,
+    targetNumberOfKnownBigLedgerPeers       = 100,
+    targetNumberOfEstablishedBigLedgerPeers = 50,
+    targetNumberOfActiveBigLedgerPeers      = 30 }
+
+
```

Comment:
It's already there: ouroboros-network#5127.

btw, your sync targets are exactly the same as the ones in [`ouroboros-network-0.21.0.0`](https://github.com/intersectmbo/ouroboros-network/blob/ouroboros-network-0.21.0.0/ouroboros-network/src/Ouroboros/Cardano/Network/Diffusion/Configuration.hs#L30-L36).